dependencies.yml

```diff
@@ -132,7 +132,7 @@ io.micrometer:
     - org.springframework:spring-webmvc
 
 io.netty:
-  netty-codec-http2: { version: &NETTY_VERSION '4.1.25.Final' }
+  netty-codec-http2: { version: &NETTY_VERSION '4.1.26.Final' }
```

Comment:
Ah was the problem just incompatibility with Netty itself?

Reply:
Yeah, it seems on. Like using wrong version of protoc with gRPC :-)